apps/portlet-documents/src/main/webapp/vue-app/attachment/components/attachments-drive-explorer/AttachmentsDriveExplorerDrawer.vue

```diff
@@ -486,6 +488,9 @@ export default {
     },
     driveExplorerDrawerTitle() {
       return this.modeFolderSelection ? this.$t('attachments.drawer.destination.folder') : this.$t('attachments.drawer.existingUploads');
+    },
+    selectedFiles() {
```

Comment:
we must not change a computed property

Reply:
more explanations: a `computed` property will be automatically refreshed by Vue and is handled by Vue lifecycle. So it's readonly properties. If you which to duplicate a property (`props`) to be able to add changes on it, yiou can add a data for `selectedFiles` + add a watch expression on `attachedFiles` to re-init `selectedFiles`. I hope it's clear enough, is it ?